jigsaw-game/client/tiles.py

```diff
@@ -0,0 +1,67 @@
+from pygame import image, sprite
+
+
+class JigSawTiles(sprite.Group):
+    """Inheritance for the addition of Group methods concerning active tiles"""
+
+    def get_active(self):
+        """Return the active tile"""
+        for tile in self.sprites()[::-1]:
+            if tile.active:
+                return tile
+        return False
+
+    def get_inactives(self):
+        """Return the inactive tiles"""
+        return [tile for tile in self.sprites() if not tile.active]
+
+
+class Tile(sprite.Sprite):
```

Comment:
This should already be typed in another PR.

Reply:
can objects be typed? or will this be referencing something else?